lambda_src/drivers/process_https.py

```diff
@@ -32,6 +33,7 @@ def process_row(
     verbose: bool = False,
     cursor: Text = '',
     results_path: Text = '',
+    write_uri: Optional[Text] = '',
```

Comment:
why this?

Reply:
This was for below, where the params get unpacked and `write_uri` is still present:  
https://github.com/Snowflake-Labs/geff/blob/d74930640267968553989195d663ca020af6955c/lambda_src/lambda_function.py#L154

But will remove `write_uri` from the params dictionary before it gets unpacked now, since it seems like the better option.